src/shared/vendor/third_party/webcomponents.js

```diff
@@ -6374,4 +6374,6 @@ CustomElements.addModule(function(scope) {
   window.Platform = scope;
 })(window.WebComponents);
 
-module.exports = window.WebComponents;
+if (typeof exports !== 'undefined') {
```

Comment:
Ugg, maybe we check for both?  It seems highly awkward to attempt to write to a variable called 'module' just because a variable called 'exports' exists (especially since the line would crash fatally if module doesn't exist).